packages/flutter/test/material/text_field_test.dart

```diff
@@ -618,11 +619,15 @@ void main() {
     await tester.pumpAndSettle();
 
     expect(controller.selection, const TextSelection(baseOffset: 0, extentOffset: 4));
-    expect(find.text('Share...'), isTargetPlatformiOS? findsOneWidget : findsNothing);
 
     if (isTargetPlatformiOS) {
+      expect(find.text('Share...'), findsOneWidget);
       await tester.tap(find.text('Share...'));
       expect(lastShare, 'Test');
+    } else {
+      expect(find.text('Share'), findsOneWidget);
+      await tester.tap(find.text('Share'));
+      expect(lastShare, 'Test');
```

Comment:
nit: consider moving `expect(lastShare, 'Test');` out of this if statement since it is common to both branches.